packages/quantic/force-app/main/default/lwc/exampleSearch/exampleSearch.html

```diff
@@ -11,6 +11,17 @@
                 </div>
                 <div class="slds-col slds-order_1 slds-large-order_2 slds-size_1-of-1 slds-large-size_9-of-12">
                     <c-quantic-search-box engine-id={engineId}></c-quantic-search-box>
+                    <ul class="slds-tabs_default slds-tabs_default__nav" role="tablist">
+                        <c-quantic-tab label="All" engine-id={engineId}></c-quantic-tab>
+                        <c-quantic-tab label="Articles" expression="@sfkbid" engine-id={engineId}></c-quantic-tab>
+                        <c-quantic-tab label="WebHelp" expression="@objecttypename==WebHelp" engine-id={engineId}></c-quantic-tab>
+                        <c-quantic-tab label="Issues" expression="@jisourcetype AND NOT @jidocumenttype=&quot;WorkLog&quot;" engine-id={engineId}></c-quantic-tab>
+                        <c-quantic-tab label="Community" expression="@objecttype==&quot;Message&quot;" engine-id={engineId}></c-quantic-tab>
+                        <c-quantic-tab label="Closed Cases" expression="@objecttype==Case AND @sfisclosed==&quot;true&quot;" engine-id={engineId}></c-quantic-tab>
+                        <c-quantic-tab label="Files" expression="@boxdocumenttype==File OR @spcontenttype==Document" engine-id={engineId}></c-quantic-tab>
+                        <c-quantic-tab label="Confluence" expression="@connectortype==Confluence2Crawler" engine-id={engineId}></c-quantic-tab>
+                        <c-quantic-tab label="Sharepoint" expression="@connectortype==SharePoint OR @connectortype==SharepointCrawler" engine-id={engineId}></c-quantic-tab>
+                    </ul>
```

Comment:
can we create a map of label and expression  for all those c-quantic-tab, we just add a foreach
What do you think?

Reply:
Yes. would be a nice way to handle this

Reply:
I would advocate to have tabs like this directly in the exampleSearch, that way people know they can just put many tabs just like the facets for example.

However, I think maybe there might be a bit too many for an example?

Reply:
I agree with @erocheleau that it would be good to have it in the html for the example. It makes it more obvious and the goal is to show how to use the components.
Even though in my own IRL implementation I might want to clean it up using @aelfaleh's suggestion.

Reply:
And also yes perhaps 5 would be a suitable number. Same as in the mockups so we know it should look visually balanced and pleasing 😄

Reply:
the tabs are from the speedbit. I knew the query expressions for those so I used that. I'll change it as per the mockup. What can I refer to get the query expressions for the tabs in the mockup :sweat_smile: